docs/source/config-file.mdx

```diff
@@ -220,3 +204,43 @@ transport:
       - mcp
       - profile
 ```
+
+## Example config file
+
+The following example file sets your endpoint to `localhost:4001`, configures transport over Streamable HTTP, provides a GraphOS key and graph reference,
+enables introspection, and provides two local MCP operations for the server to expose.
+
+```yaml config.yaml
+endpoint: http://localhost:4001/
+transport:
+  type: streamable_http
+graphos:
+  apollo_key: <YOUR_APOLLO_KEY>
```

Comment:
Not sure who would want to set the API key directly in a config file. 🤔 I feel most folks would prefer using an environment variable instead.

Reply:
That's a good point, will remove!